ui/revenuecatui/src/main/res/values/strings.xml

```diff
@@ -18,4 +18,6 @@
     <string name="monthly">Monthly</string>
     <string name="weekly">Weekly</string>
     <string name="lifetime">Lifetime</string>
+    <string name="no_browser_cannot_open_terms">No browser installed. Terms and conditions could not be opened.</string>
+    <string name="no_browser_cannot_open_privacy">"No browser installed. Privacy policy could not be opened."</string>
```

Comment:
Let me know what you think of this copy. I'll add translations after a second pair of eyes gives the thumbs up 🙂